src/redux/slices/editingProject.ts

```diff
@@ -50,7 +50,7 @@ export const defaultProjectState: EditingProjectState = {
       twitter: '',
       discord: '',
       tokens: [],
-      version: 4,
+      version: 5,
```

Comment:
**suggestion** lets make a const for this and use it everywhere pls #singlesourceoftruth